readme.txt

```diff
@@ -39,7 +39,7 @@ When opening a bug on GitHub, please give us as many details as possible.
 
 == Changelog ==
 
-= 2.5.0 - 2021-xx-xx =
+= 2.5.0 - 2021-05-19 =
```

Comment:
I think we normally duplicate the changelog in the `readme.txt`...

Reply:
Ooh good point - that might be a missing step? This has already gone to WPORG.

Does this matter - does it affect anything, if we have a separate changelog? Maybe we can delete the changelog content from readme.

Let me know what you think, I can paste over the changelog if this needs to be complete.

Reply:
Sadly, the changelog from `readme.txt` is currently visible here: https://wordpress.org/plugins/facebook-for-woocommerce/#developers

It might also be shown in the plugin admin view when updating...

Reply:
Right - sounds like we need to keep it updated then! I'll add this to process. 

I'll investigate pushing up just that change to WPORG so it shows all the changes. Thanks for catching this!